uint8 id
uint16 position
uint16 ms
uint16 speed
----
uint8 status
